backend/kbsb/tst_endpoints.py

```
@@ -16,7 +16,14 @@
 def hello():
     return "hello world"
 
+
 @app.get("/api/testmail", include_in_schema=False)
 def hello():
```

Ruff (0.14.0)

Redefinition of unused `hello` from line 16

(F811)